driver/main.c

```diff
@@ -2941,6 +2928,43 @@ MODULE_INFO(build_commit, DRIVER_COMMIT);
 MODULE_INFO(api_version, PPM_API_CURRENT_VERSION_STRING);
 MODULE_INFO(schema_version, PPM_SCHEMA_CURRENT_VERSION_STRING);
 
+static int set_g_per_cpu_buffer_dim(const char *val, const struct kernel_param *kp)
```

Comment:
This is beautiful!

Reply:
That's a smart idea but probably I would keep the overall buffer dim just to avoid another way to indicate the same thing: number_of_pages, overall_buffer_dim, power_of_two